### PR TITLE
Update DG for changes to model component

### LINKED ISSUE #87: Update DG for changes to model component
Let's update the DG for changes made to the model component.
Related to #6

### PR DESCRIPTION
**Pull request by aureliony**
Closes #87

### CODE REVIEW

File: docs/diagrams/ModelClassDiagram.puml [hunk 1]
```diff
@@ -36,15 +39,16 @@ ModelManager -right-> "1" UserPrefs
 UserPrefs .up.|> ReadOnlyUserPrefs
 
 AddressBook *--> "1" UniquePersonList
+UniquePersonList .right-> UniquePersonListDeserializer
```
Comment (bryanyee33) [inline docs/diagrams/ModelClassDiagram.puml:42]: Should UniquePersonListDeserializer be included? It seems like a less-important class that can be omitted for comprehensibility.

**Review by bryanyee33 — COMMENTED** [review]
Just a minor nit.

File: docs/diagrams/ModelClassDiagram.puml (hunk 1 above)
Comment (aureliony) [reply docs/diagrams/ModelClassDiagram.puml]: Removed.

**Review by yisiox — APPROVED** [review]
LGTM

**Pull request merged**